syntax_def_dev.py

```diff
@@ -508,3 +508,77 @@ def inhibit(ret):
         ]
 
         return inhibit(self.base_completions + completions)
+
+class SyntaxDefRegexCaptureGroupHighlighter(sublime_plugin.ViewEventListener):
+    @classmethod
+    def is_applicable(cls, settings):
+        syntax = settings.get('syntax')
+        return syntax == 'Packages/PackageDev/Package/Sublime Text Syntax Definition/Sublime Text Syntax Definition.sublime-syntax'
+
+    def on_selection_modified(self):
+        self.view.erase_regions('captures')
```

Comment:
it would be better to only erase the regions when they no longer need to be drawn - i.e. the selection/carets move out of the captures area - to avoid flickering. This is how the syntax test highlighting works.